packages/flutter/test/material/about_test.dart

```diff
@@ -41,31 +43,45 @@ void main() {
 
     expect(find.text('About Pirate app'), findsNothing);
     expect(find.text('0.1.2'), findsNothing);
+    expect(find.byWidget(logo), findsNothing);
+    expect(find.text('I am the very model of a modern major general.'),
```

Comment:
See Flutter style guide on how to style this: 
- [Prefer max of 80 characters](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-a-maximum-line-length-of-80-characters)
- [Indent multi-line args and param list by 2 characters](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#indent-multi-line-argument-and-parameter-lists-by-2-characters)

My preference would be: 
```
    expect(
      find.text('I am the very model of a modern major general.'),
      findsNothing,
    );
```
However, an alternative you can do if it goes just slightly over the 80 char limit:
```
    expect(find.text('I am the very model of a modern major general.'), findsOneWidget);
```